app/src/main/java/com/hover/stax/domain/use_case/financial_tips/GetTipsUseCase.kt

```diff
@@ -1,11 +1,23 @@
 package com.hover.stax.domain.use_case.financial_tips
 
 import com.hover.stax.domain.model.FinancialTip
+import com.hover.stax.domain.model.Resource
 import com.hover.stax.domain.repository.FinancialTipsRepository
-import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.*
```

Comment:
Would it make sense to make specific imports rather than wildcard imports? Wildcard imports over time just impacts on complie time

Reply:
I'm gonna have to change the IDE settings to effect this.